ui/app/AppLayouts/Shell/ShellContainer.qml

```diff
@@ -51,7 +51,8 @@ Control {
     spacing: Theme.bigPadding
 
     function focusSearch() {
-        searchField.forceActiveFocus()
+        // Need to use Qt.callLater to ensure the focus is set after the component is fully loaded
+        Qt.callLater(() => searchField.forceActiveFocus())
```

Comment:
Since the Shell is shown at the start for real now (loaded by the main module), it loaded too fast and the focus didn't apply